cads_adaptors/tools/url_tools.py

```diff
@@ -25,34 +29,40 @@ def __init__(
         target: str,
         maximum_retries: int,
         retry_after: float | tuple[float, float, float],
+        tqdm_kwargs: dict[str, Any] = {},
```

Comment:
Can you instatiate a `None`. We discovered the hard way that dask sometimes preserves the mutable kwargs from process/request to process/request.